crates/turborepo-analytics/src/lib.rs

```diff
@@ -1,5 +1,10 @@
 #![deny(clippy::all)]
 
+//! Turborepo's analytics library. Handles sending analytics events to the
+//! Vercel API in the background. Currently we only record cache usage events,
```

Comment:
> Currently we only...

I'd recommend not writing point-in-time docs. They are out of date the minute they ship because "currently" doesn't mean anything when you're reading the doc.